keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -491,37 +456,23 @@ Defined using feature gate
 - Plaintext output roughly matches explain output
 - OpenAPIV3 (raw json) output implemented
 
-#### Alpha 2
-
-If we decide to move ahead with the `md` and `html` outputs, an Alpha 2 may
-be required.
-
-- `md` output implemented (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Section for each individual GVK
-  - All types hyperlink to specific section
--  basic `html` output  (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Page for each individual GVK.
-  - All types hyperlink to their specific page
-  - Searchable by name, description, field name.
-
 #### Beta
 
-- kube-openAPI v3 JSON deserialization is optimized to take less than 150ms on
-  most machines
 - OpenAPI V3 is enabled by default on at least one version within kubectl's support window.
 As of Kubernetes 1.24 OpenAPIV3 entered beta and become enabled by default, therefore meeting this requirement.
+In Kubectl for release 1.25, all k8s versions within support window will be able to have OpenAPIV3 enabled.
+However, the fallback is kept around since it may not always be enabled.
+
 - `--output plaintext` is on-by-default and environment variable is removed/on by default
```

Comment:
This makes the feature on by default as stated, and in "Version Skew Strategy" below we state that "OpenAPIV3 is Beta as of Kubernetes 1.24. This feature should not be on-by-default until it is GA.". It looks like it is proposed for GA in this release: https://github.com/kubernetes/enhancements/pull/3773. Does that mean this PR effectively depends on that one?

Reply:
"This feature should not be on-by-default until it is GA" should be expanded upon to say we should not remove fallback until OpenAPIV3 is GA for all versions in skew. I'll update the KEP.

This PR does not depend on OpenAPIV3 being GA, since we will have a fallback mechanism and way to positively choose to use openapiv2.